### PR TITLE
Manilka Dilshan

### PR DESCRIPTION
**Pull request by ManilkaDilshan**

### CODE REVIEW

**Comment by ManilkaDilshan** [comment]
new PR

File: index.js [hunk 1]
```diff
@@ -4,9 +4,32 @@ const clearButton = document.getElementById("clear");
 const list = document.getElementById("un-ordered-list");
 const input = document.getElementById("input");
 
-addButton.addEventListener("click", () => {});
-removeButton.addEventListener("click", () => {});
+addButton.addEventListener("click", () => {
+  const userInput = input.value.trim();
+
+  if (userInput !== "") {
```
Comment (lakindu2002) [inline index.js:10]:
```suggestion
  if (userInput.length == 0) {
```

File: index.js [hunk 2]
```diff
@@ -4,9 +4,32 @@ const clearButton = document.getElementById("clear");
...
+    lastItem.remove();
+  }
+});
+
 
 clearButton.addEventListener("click", () => {
-  list.innerHTML = "";
+  setTimeout(() => {
+    list.innerHTML = "";
+  }, 2000);
```
Comment (lakindu2002) [inline index.js:34]:
```
console.log("1")
 setTimeout(() => {
    list.innerHTML = "";
    console.log("2")
  }, 2000);
console.log("3")
```

Comment (lakindu2002) [reply index.js:34]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Event_loop

**Pull request closed without merging**